synapse/events/third_party_rules.py

```diff
@@ -78,11 +78,12 @@ def on_create_room(self, requester, config, is_requester_admin):
         """
```

Comment:
It would be good to update the docstring here to say that `defer.Deferred[bool]` should be returned:

```
Returns:
    defer.Deferred[bool]: Whether room creation is allowed or denied.
```